### PR TITLE
Display like features

### PR DESCRIPTION
**Pull request by akos29**
### Display the number of likes for each movie

In this request majorly incorporated: 

- Add a module for Involvement API 
- Add functionality to match the number of likes with the movie from the base API
- Update the populated movies with a dynamic number of likes

This will resolve issue #12

### CODE REVIEW

File: src/modules/updateLikes.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import * as el from './domElements.js';
+import Involve from './involvementApi.js';
+
+export const updateLikes =  (name) => { 
+    const status =Involve.postLike(name,1);
+    console.log("updateLikes: ",status );
+    return status;
```
Comment (krisbobo) [inline src/modules/updateLikes.js]: Consider removing the `console.log()` used; as this does not match JS best practices.

File: src/modules/involvementApi.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    static displayLikes= async() =>{
+        const response = await fetch(`${this.BASE_URL}/likes`);
+        const likes = await response.json();   
+        // console.log(likes)
+        return likes;
+    }
+    static displayComments= async(id) =>{
+        // const response = await fetch(`${this.BASE_URL}/comments?item_id=item_id=The Fourth Hand`);
+        const response = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/rFc4tlCMcxbVijXIn6ie/comments?item_id=The%20Fourth%20Hand`);
```
Comment (krisbobo) [inline src/modules/involvementApi.js]: To improve your code's readability, I suggest you remove commented pieces of old code.

File: src/modules/involvementApi.js [hunk 3]
```diff
@@ -1,69 +1,32 @@
 export default class Involve {
-    static BASE_URL =`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${process.env.API_KEY}`;
-    constructor(){
+  static BASE_URL =`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${process.env.API_KEY}`;
```
Comment (krisbobo) [inline src/modules/involvementApi.js:3]: Great job securing the id. This improves the overall security of your app.

File: .env [hunk 4]
```diff
@@ -0,0 +1 @@
+API_KEY=rFc4tlCMcxbVijXIn6ie
```
Comment (krisbobo) [inline .env:1]: Consider adding the key to the `.gitignore` file upon the approval of this PR

**Review by krisbobo — APPROVED** [review]
# Status: Approved✅✅✅
Hi @akos29
You did a great job in implementing the requirements for this block of work 
## Highlights:
- Linters✔️
- Project Requirements✔️
- Best Practice✔️
### Next Action:
There's nothin else to say other than; it is time to merge this PR. Consider implementing the suggested changes below so as to make your application more robust. they do not hinder the approval of this PR.

File: src/modules/updateLikes.js (hunk 1 above)
Comment (akos29) [reply src/modules/updateLikes.js]: Thanks! Updated accordingly 😉

**Pull request merged**